### DISCUSSION TITLE
Import local repository

### DISCUSSION THREAD

**Discussion by simoneceruti-schiavi** [community/community#46712] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi all, I have this problem:
I need to import a repository that is present in an our internal server.
I tried but the import not success, probably due to firewill and or right access from external.
Is there the possibility to import it so that at end I have the repository on github server (with history too).
If yes, how?
Thank
Ceruti

**Comment by jge162** [comment]
@simoneceruti-schiavi  yes, got to '+' on top right corner of github on page on a computer only. then click import repository. enter the url and your going to setup all the following things you want, private, public etc. click begin import. see screen shots below. 
<img width="195" alt="Screenshot 2023-02-08 at 12 31 29 AM" src="https://user-images.githubusercontent.com/31228460/217476524-f9d7ad4f-12da-4576-b800-15098b817719.png">
<img width="805" alt="Screenshot 2023-02-08 at 12 31 38 AM" src="https://user-images.githubusercontent.com/31228460/217476541-0ab4c7f0-fb8f-4872-a1f8-088b188bc4e4.png">

**Reply by simoneceruti-schiavi** [reply]
I tried but I have this error:
![image](https://user-images.githubusercontent.com/124770443/217480604-0ca24b20-bd63-4871-8208-81bcfaa8e87d.png)
Suggestions?
Thanks

**Reply by jge162** [reply]
Github only supports importing repositories via HTTPS and SSH, make sure the URL you are using is in one of these formats. You cannot use a network file path

**Comment by simoneceruti-schiavi** [comment]


**Reply by simoneceruti-schiavi** [reply]
Is it possible to copy the repository using another way and not standard functions?
Eventually we can pay this service.
If yes how should I contact?
Thanks
